package.json

```diff
@@ -13,9 +13,9 @@
   "readmeFilename": "README.md",
   "gitHead": "eb6f38decdf15fc18625bf60a602db2937feb229",
   "scripts": {
-    "build": "$(npm bin)/grunt",
-    "release": "$(npm bin)/grunt release",
-    "watch": "$(npm bin)/grunt watch"
```

Comment:
NPM scripts run with `./node_modules/.bin` added to the PATH, so it will use the locally installed versions automatically.